parser_library/src/processing/preprocessor.h

```diff
@@ -81,8 +81,9 @@ class preprocessor
 
     static bool is_continued(std::string_view s);
 
-    void do_highlighting(
-        const semantics::preprocessor_statement_si& stmt, semantics::source_info_processor& src_proc) const;
+    virtual void do_highlighting(const semantics::preprocessor_statement_si& stmt,
+        semantics::source_info_processor& src_proc,
+        size_t continue_column = 15) const;
```

Comment:
continue_column seems really wrong here, why should the caller know the right value, without even knowing the underlaying type?
maybe protected do_common_highlighting? and then void do_highlighting(...){do_common_highlighting(..., 15);}?

Reply:
method is indeed protected. my mistake.